extensions/2.0/Khronos/KHR_XMP/README.md

```diff
@@ -0,0 +1,141 @@
+# KHR_XMP
```

Comment:
I would prefer `KHR_xmp` instead here. There are two existing Khronos precedents I'm aware of:

- In WebGL, extensions may be preceeded by a browser vendor prefix, such as `WEBKIT_OES_vertex_array_object`.
- Otherwise, acronyms are almost universally lowercase (`WEBGL_compressed_texture_pvrtc`, `WEBGL_compressed_texture_s3tc`,  `WEBGL_compressed_texture_etc`, `WEBGL_compressed_texture_astc`,  `EXT_shader_texture_lod`) with the notable  exceptino of `EXT_sRGB`.

Because `XMP` is not a prefix (e.g. we don't expect to have separate extensions for each XMP namespace), it seems more idiomatic to make it lowercase.

Reply:
I agree, I will change it to `KHR_xmp` if there are no objections. Thanks for pointing that out.